script/deploy/safe/confirm-safe-tx.ts

```diff
@@ -198,11 +229,17 @@ const func = async (network: string, privateKey: string, rpcUrl?: string) => {
       consola.success('Transaction signed', tx.safeTxHash)
     }
 
+    if (action === 'Sign') {
+      await signTransaction(txToConfirm)
+    }
+
+    if (action === 'Sign & Execute Now') {
+      await signTransaction(txToConfirm)
+      await executeTransaction(txToConfirm)
+    }
+
```

Comment:
So I think there may be an issue unless you've tested this already. I found that when signing all the way through the transaction is no longer in the 'pending' queue and won't be returned from the API.

Reply:
For me this worked fine: 

![image](https://github.com/user-attachments/assets/dc83ff92-814d-4760-bf66-9f07f4f6764c)

I will now propose the missing facet for linea so you can test the same on your end, ok? but you need to confirm the tx from this branch to use the updated code.

Reply:
@ezynda3 can we merge this pls?